### PR TITLE
Support freezing modules for windows ports

### PR DESCRIPTION
**Pull request by stinos**

### CODE REVIEW

File: tools/make-frozen.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def module_name(f):
 for f, st in modules:
     print("%d," % st.st_size)
 
-print("};")
+print("0};")
```
Comment (dlech) [inline tools/make-frozen.py:53]: Is this supposed to be `\\0`?

Comment (stinos) [reply tools/make-frozen.py:53]: No this is for the `uint32_t mp_frozen_str_sizes` array

Comment (dlech) [reply tools/make-frozen.py:53]: Ah, I see it now.

**Comment by dpgeorge** [comment]
Thanks!  Merged in 4b35aa5730cf26992453e3ce7cf74ced4128fe20 though 5b94c610971334019e7301026e0fbbf5950e094d

Note re the freezing tools: would be good to eventually combine str/mpy freezing capabilities into one (instead of having 2 separate arrays) but for now the fix here for zero-length arrays is good.

**Pull request closed without merging**